cove_360/lib/threesixtygiving.py

```diff
@@ -389,7 +389,7 @@ def process(self, grant, path_prefix):
             count_failure = False
             for num, organization in enumerate(grant['recipientOrganization']):
                 has_postal_code = organization.get('postalCode')
-                has_location_data = organization.get('location') and organization.get('location').get('geoCode') and organization.get('location').get('geoCodeType')
+                has_location_data = organization.get('location') and any(location.get('geoCode') and location.get('geoCodeType') for location in organization.get('location'))
                 complete_recipient_org_data = has_postal_code and has_location_data
```

Comment:
Yes, that's right? Do you want to make these changes to the `800-802-fix-additional-checks` branch?